src/modules/tasks.js

```diff
@@ -0,0 +1,211 @@
+  // const taskForm = document.createElement('form');
+  // taskForm.className = "taskForm border border-dark px-2 mx-5 ";
+  // taskForm.setAttribute('id', 'taskForm');
+  if (!countForm) {
+    taskForm.innerHTML = `
+    <input type='text' class=' my-2' name='taskName' id='task-name' placeholder='Add title here'>
+    <textarea  id='comment' class='my-2' rows='3' maxlength='150' placeholder='Add comment here'></textarea>
+    <select class='form-select me-4' id='selectPriority' aria-label='Default select example'>
+            <option selected value='1'>Select priority level</option>
+            <option class='one' value='1'>&#xf024; Priority 1</option>
+            <option class='two' value='2'>&#xf024; Priority 2</option>
+            <option class='three' value='3'>&#xf024; Priority 3</option>
+            <option class='four' value='4'>&#xf024; Priority 4</option>
+    </select>
+    <input type="date" max='2110-13-13' id='datePicker'>
+    <button type='button' class='confirmTask btn btn-secondary' id='confirmTask'>Create task</button>
+    <button type='button' class='dismissTask btn btn-light m-3' id='dismissTask'>Dismiss</button>
+  `;
+
+    document.getElementById('datePicker').setAttribute('min', setDate());
+    countForm = true;
+  } else {
+    taskForm.innerHTML = '';
+    countForm = false;
+  }
+};
+
+const changeStatus = (index) => {
+  // eslint-disable-next-line
+  myTasks[index].finished ? myTasks[index].finished = false : myTasks[index].finished = true;
+};
+
+const deleteAllTasks = (currentProject) => {
+  const arr = myTasks.filter((task) => task.projectsN !== currentProject);
+  myTasks = arr;
+  first.innerHTML = '';
+  tableArticle.innerHTML = '';
+  todayContainer.innerHTML = '';
+};
+
+const makeEditForm = (task) => {
+  editTaskForm.innerHTML = `
+      <input type='text' class=' my-2' name='EdiTaskName' id='edit-task-name' value='${task.title}'>
+      <textarea  id='editComment' class='my-2' rows='3' maxlength='150'> ${task.comment}</textarea>
+      <select class='form-select me-4' id='selectPriority1' aria-label='Default select example'>
+              <option selected value='1'>Priority ${Flag(task.priority)}</option>
+              <option class='one' value='1'>&#xf024; Priority 1</option>
+              <option class='two' value='2'>&#xf024; Priority 2</option>
+              <option class='three' value='3'>&#xf024; Priority 3</option>
+              <option class='four' value='4'>&#xf024; Priority 4</option>
+      </select>
+      <input type="date" max='2110-13-13' id='datePicker1' value='${task.date}'>
+      <button type='submit' class='confirmEditTask btn btn-secondary' id='confirmEditTask'>Edit task</button>
+      <button type='button' class='dismissEditTask btn btn-light m-3' id='dismissEditTask'>Dismiss</button>
+    `;
+
+  document.getElementById('datePicker1').setAttribute('min', setDate());
+};
+
+const makeToday = () => {
+  todayContainer.innerHTML = '';
+  // eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files